includes/admin/downloads/metabox.php

```diff
@@ -440,7 +440,12 @@ function edd_render_price_row( $key, $args = array(), $post_id, $index ) {
 			$actions['show_advanced'] = '<a href="#" class="toggle-custom-price-option-section">' . __( 'Show advanced settings', 'easy-digital-downloads' ) . '</a>';
 		}
 
-		$actions['remove'] = '<a class="edd-remove-row edd-delete" data-type="price">' . sprintf( __( 'Remove', 'easy-digital-downloads' ), $key ) . '<span class="screen-reader-text">' . sprintf( __( 'Remove price option %s', 'easy-digital-downloads' ), $key ) . '</span></a>';
+		$actions['remove'] = sprintf(
+			'<a aria-label="%s" class="edd-remove-row edd-delete" data-type="price">%s</a>',
+			/* translators: %s: download price option */
+			sprintf( __( 'Remove price option %s', 'easy-digital-downloads' ), $key ),
+			__( 'Remove', 'easy-digital-downloads' )
```

Comment:
See other note about escaping the replaced value.

We should update these to include `esc_html()` and `esc_attr()` as well.